addons/website_event_track/controllers/event_track.py

```diff
@@ -24,13 +24,14 @@ def _get_event_tracks_base_domain(self, event):
         """ Base domain for displaying tracks. Restrict to accepted or published
         tracks for people not managing events. Unpublished tracks may be displayed
         but not reachable for teasing purpose. """
-        search_domain_base = [
+        search_domain_base = ['&',
             ('event_id', '=', event.id),
+            ('stage_id.is_visible_in_agenda', '=', True),
```

Comment:
Why ? Shouldn't event manager see all tracks in agenda while working on it ? That was the reasoning behind that I think.

Reply:
Luc points out that the agenda can quickly get bloated with tracks. To reduce the payload of the agenda, Luc wants the tracks linked to a track stage having `is_visible_in_agenda` set to `False` to be hidden in the agenda. To ensure consistency between the list view and the agenda, I also decided to hide those tracks from the list view (`/event/<event-id>/track`. Hence, `is_visible_in_agenda` impacts the visibility of the tracks in the frontend but it has some differences compare to `is_published`:

- If the track is linked to a track stage having `is_visible_in_agenda` set to `False`, the track will not be visible in the frontend (neither for the visitors nor for the administrators/event managers).
- If the track is not published (i.e: when the administrators clicks on the 'published' toggle of the Odoo navbar), the track will be visible for the administrators/event managers only. On the frontend, those tracks will have a small badge telling the administrators that the element is not published.